main.go

```diff
@@ -782,9 +795,20 @@ func postCollaboratorPageHandler(c *gin.Context, db *sql.DB) {
 	}
 
 	repoId := c.PostForm("repo_id")
```

Comment:
remove this field. Instead of getting it from form. Derive it from repo_name param (get current repo).

Reply:
DONE